templates/inactive_revision.html

```diff
@@ -0,0 +1,33 @@
+<p>
+    The following {{ plural('bug has', data, pword='bugs have') }} inactive patches that need action from either the author or a reviewer:
+</p>
+<table {{ table_attrs }}>
+    <thead>
+        <tr>
+            <th>Bug</th>
+            <th>Summary</th>
+            <th>Patches</th>
```

Comment:
Could you also add a column that mentions who's going to be needinfoed? It'd make it easier to check the dry-run looks good

Reply:
Done here: e34cd1f2fae346e4552eaa9c8bac1f927249633b